### PR TITLE
Update portability-analyzer.md

### PR DESCRIPTION
**Pull request by brandonh-msft**
## Summary

Propagating [the notice from the repo's README regarding deprecation](https://github.com/microsoft/dotnet-apiport#net-api-portability) into this landing page for its aka.ms link

### CODE REVIEW

File: docs/standard/analyzers/portability-analyzer.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ ms.assetid: 0375250f-5704-4993-a6d5-e21c499cea1e
 
 # The .NET Portability Analyzer
 
+> ***Note:** We're in the process of deprecating API Port in favor of integrating binary analysis directly into [.NET Upgrade Assistant](https://github.com/dotnet/upgrade-assistant). In the upcoming months, we're going to shutdown the backend service of API Port which will require to use the tool in offline mode. The instructions to use API Port in offline mode can be found [here](https://github.com/microsoft/dotnet-apiport/blob/dev/docs/Console/README.md#run-the-tool-in-an-offline-mode).*
```
Comment (IEvangelist) [inline docs/standard/analyzers/portability-analyzer.md:10]:
```suggestion
> [!NOTE]
> We're in the process of deprecating API Port in favor of integrating binary analysis directly into [.NET Upgrade Assistant](https://github.com/dotnet/upgrade-assistant). In the upcoming months, we're going to shut down the backend service of API Port, which will require using the tool offline. For more information, see [GitHub: .NET API Port repository](https://github.com/microsoft/dotnet-apiport/blob/dev/docs/Console/README.md#run-the-tool-in-an-offline-mode).*
```

**Review by IEvangelist — COMMENTED** [review]
Sounds good, just a suggestion to update the text a bit and convert into an alert.

File: docs/standard/analyzers/portability-analyzer.md (hunk 1 above)
Comment (brandonh-msft) [reply docs/standard/analyzers/portability-analyzer.md]: thanks!

Comment (brandonh-msft) [reply docs/standard/analyzers/portability-analyzer.md]: not sure if that ending `*` is required for some reason so re-requesting your 👀

**Review by terrajobst — APPROVED** [review]

**Pull request merged**